bzkanban.js

```diff
@@ -1362,6 +1362,7 @@ function showBugModal(bugCurrent, bugUpdate) {
     meta.className = 'bug-meta';
 
     body.appendChild(comments);
+    showSpinner();
```

Comment:
Can be reverted, it was related to the discussion about slowness and I was adding some type of user notification. https://github.com/leif81/bzkanban/pull/57#issuecomment-276247231

In fact, could be switched out for a spinning cursor